<launch>
  <!--Node for teleop control callbacks-->
  <rosparam command="load" file="$(find mrover)/config/teleop.yaml" />
  <node name="jetson_teleop" pkg="mrover" type="jetson_teleop.py"/>
  <!--
    Run the drive program
  -->
  <node name="odrive_control" pkg="mrover" type="odrive_control.py" output="screen"></node>
  <rosparam command="load" file="$(find mrover)/config/esw.yaml" />
  <!--
    Operate Jetson cameras for all missions.
  -->
  <node name="cameras" pkg="mrover" type="cameras.py"></node>
</launch>